cpp/example_code/dynamodb/batch_get_item.cpp

```diff
@@ -83,12 +80,9 @@ int main(int argc, char** argv)
 		// Table2: ProductCatalog.
 		Aws::String t2Name = "ProductCatalog";
 		Aws::DynamoDB::Model::KeysAndAttributes t2KeyAttrs;
-		//Table2: Projection expression
-		t2KeyAttrs.SetProjectionExpression("Title, Price, Color");
+		2KeyAttrs.SetProjectionExpression("Title, Price, Color");

```

Comment:
Did you mean to change the name of this variable? I think it'll break the code.

Reply:
fixed